modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -47,6 +47,19 @@ $ oc patch argocd openshift-gitops \
 -n openshift-gitops --type=merge \ 
 --patch-file out/argocd/deployment/argocd-openshift-gitops-patch.json
 ----
++
+[NOTE]
+====
+For a disconnected environment, you must patch the `out/argocd/deployment/argocd-openshift-gitops-patch.json` file with the `ztp-site-generator` image mirrored in your local registry by running the following command:
+[source,terminal]
+----
+$ oc patch argocd openshift-gitops -n openshift-gitops --type='json' \
+-p='[{"op": "replace", "path": "/spec/repo/initContainers/0/image", \
+"value": "${LOCAL_REGISTRY}/${ZTP_SITE_GENERATOR_IMAGE_REF}"}]'
+----
+
+In this example, `LOCAL_REGISTRY` represents the URL to the disconnected registry and `ZTP_SITE_GENERATOR_IMAGE_REF` is the path to the mirrored `ztp-site-generator` image in the local registry.
```

Comment:
There is a "where `<variable>`" construction that is useful in this scenario. You can see an example of it [here](https://docs.openshift.com/container-platform/4.13/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html#cnf-performing-end-to-end-tests-running-the-tests_cnf-latency-tests).

```suggestion
----
+
where:
+
--
<LOCAL_REGISTRY>:: Is the URL of the disconnected registry, for example, `my.local.registry:5000`
<ZTP_SITE_GENERATOR_IMAGE_REF>:: Is the path to the mirrored `ztp-site-generator` image in the local registry, for example `openshift4-ztp-site-generate:custom`.
--
In this example, `LOCAL_REGISTRY` represents the URL to the disconnected registry and `ZTP_SITE_GENERATOR_IMAGE_REF` is the path to the mirrored `ztp-site-generator` image in the local registry.
```